boxsdk/object/base_object.py

```diff
@@ -34,9 +34,9 @@ def _description(self):
         """Base class override.  Return a description for the object."""
         if 'name' in self._response_object:
             return '{0} ({1})'.format(self._object_id, self.name)  # pylint:disable=no-member
-        else:
-            return '{0}'.format(self._object_id)
+        return '{0}'.format(self._object_id)
 
+    # pylint:disable=arguments-differ
```

Comment:
if it's just for this method, please put the pylint directive inside the method, like so:

    def get_url(self, *args):
        """Docstring"""
        # pylint:disable---